plugin-flex-ts-template-v2/src/feature-library/send-audio-rec-file/README.md

```diff
@@ -0,0 +1,31 @@
+# send-audio-rec-file
+This is a feature that enables agents on Flex to record, listen and send audio messages on chat task. With this feature, agents can record and send audio messages to customers.
+
+
+# flex-user-experience
+The audio message feature provides a seamless user experience for agents on Flex. Agents can easily access the record panel feature by clicking on a button that enables them to record and listen the audio message. The feature is designed to work across chat channels, including WhatsApp and Web Chat, and admin can choose to enable or disable audio message sending feature based on their preferences.
+
+![sendAudioFeature](https://github.com/rickmachadotwilio/flex-project-template/assets/104230389/1631bfbe-930a-435f-9a78-769971818598)
+
+
+
+# setup and dependencies
+To use the audio message feature, you need to have access to the Twilio Flex platform. You also need to ensure that your Flex account is set up correctly, and you have the necessary permissions as agent or supervisor.
+
+The audio message feature has the following dependencies:
+
+* A browser that supports audio recording and playback, such as Chrome, Firefox, or Safari.
+* An internet connection with adequate bandwidth to transmit audio messages.
+
+
+# how does it work?
```

Comment:
this section is really to describe whats happening under the hood.
something like, the new component is rendered that captures audio as a fine and is pushed to the conversation as a binary file using X api.  The conversation channel is then responsible for pushing that binary file down to each participant on their specific channel interface (whatsapp, sms, webchat) and then the client for that channel is responsible for recognizing the binary file as an audio file and allow the user to play it.... 

or something like that...